src/main/java/org/jenkinsci/plugins/ewm/actions/ExwsAllocateAction.java

```diff
@@ -0,0 +1,46 @@
+package org.jenkinsci.plugins.ewm.actions;
+
+import hudson.model.Action;
+import org.jenkinsci.plugins.ewm.Messages;
+import org.jenkinsci.plugins.ewm.steps.model.ExternalWorkspace;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+
+/**
+ * {@link Action} implementation for the {@link org.jenkinsci.plugins.ewm.steps.ExwsAllocateStep}.
+ * Stores the allocated {@link ExternalWorkspace}.
```

Comment:
I think the doc should say, here or somewhere else, why we store the ext workspace, e.g. for later use in downstream jobs, for removal by another job, etc.